testnets/sedatestnet/chain.json

```diff
@@ -58,7 +58,7 @@
         "version": "v0.50.13"
       },
       "cosmwasm": {
-        "version": "0.53.2",
+        "version": "0.53.3",
```

Comment:
Looks like this has to be v0.53.3 to match the version in the "versions.json" file

`Path: cosmwasm.version, Reason: Value mismatch, In "codebase": 0.53.3, In currentVersion: v0.53.3`